lib/pbench/cli/agent/commands/results/__init__.py

```diff
@@ -58,42 +56,42 @@ def move_results(ctx, _user, _prefix, _show_server):
         logger.error("Failed to create temporary directory")
         sys.exit(1)
 
-    dirs = [
-        _dir
-        for _dir in next(os.walk(config.pbench_run))[1]
-        if not _dir.startswith("tools-") and not _dir.startswith("tmp")
-    ]
+    runs_copied = 0
+    failures = 0
```

Comment:
Probably pointless to repeat it here, but I don't see any use of `failures` ... so it seems that any copy failures go unreported? If that's deliberate, we should drop `failures` from the return tuple and just return the success count.

Reply:
Answered in PR #1947.